test/api/dto/munkatars.ts

```diff
@@ -0,0 +1,11 @@
+export class Munkatars {
```

Comment:
domains and file names were English so far, I would vote for keeping it that way for consistency. eighter way of mixing the two is suboptimal in my opinion in the same file.

Reply:
True!